src/style.css

```diff
@@ -0,0 +1,76 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.searchBar {
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  /* box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1); */
+  color: #3d4e61;
+  margin-top: 20px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  cursor: pointer;
+}
+
+.searchBar-big {
+  height: 70px;
+  background-image: url("images/Search.svg");
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+  background-repeat: no-repeat;
+  padding: 24px 100px 24px 62px;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+}
+
+.searchBar-small {
+  height: 42px;
+  background-image: url("images/Search.svg");
+  /* scale backgrownd image */
+  background-size: 11px 11px;
+  background-position: 12px 14px;
+  background-repeat: no-repeat;
+  padding: 10px 100px 11px 33px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 19px;
+}
+
+.searchBar::placeholder {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  color: #3d4e61;
+}
+
+.searchBar-big::placeholder {
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.searchBar-small::placeholder {
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.searchBar:hover {
+  border: 1px solid #8f00ff;
```

Comment:
hover and focus states should match the layout (there is no such colors)
<img width="622" alt="Screenshot 2023-03-20 at 20 11 53" src="https://user-images.githubusercontent.com/117587463/226429677-4dbf43d6-30b9-4103-a6fe-394858680d6d.png">

Reply:
Good afternoon. I am not sure about colors. I chose fiolet for hover and aqwa blue for focus. May be I should shoose  darker color shade, for example dark violet #9400d3 https://icolorpalette.com/color/dark-violet and blue cola #0085e5 https://icolorpalette.com/color/0085e5. Who knows? If a person is a colorblind what he will do.  I guess a disigner should give information  about hex code.

Reply:
I replaced colors with the above ones.

Reply:
there are no brightly colors at the given mockup 
<img width="787" alt="Screenshot 2023-03-21 at 20 28 41" src="https://user-images.githubusercontent.com/117587463/226706927-e0fa18d0-2cb5-4b85-b19b-ffdfe6789df3.png">